tools/converters/excel/appsettings.json

```diff
@@ -59,6 +59,10 @@
               {
                 "Search": "Insurance Coporation of BC",
                 "Value": "Insurance Corporation of BC"
+              },
+              {
+                "Search": "LNG, Crown Land Opportunities and Restoration",
+                "Value": "Crown Land Opportunities"
```

Comment:
Needed to update the Converter Tool so that future generated files will contain the correct agency name.